tests/envs/test_clear_and_place_env.py

```diff
@@ -19,11 +21,30 @@
 def test_clear_and_place_env():
     """Tests for ClearAndPlacePyBulletBlocksEnv()."""
 
-    scene_description = ClearAndPlaceSceneDescription(
+    # For the sake of this test with hardcoded motion, force the block to start
+    # out in a "safe" location where the pushing shouldn't impact it at all.
+
+    @dataclass(frozen=True)
+    class _CustomClearAndPlaceSceneDescription(ClearAndPlaceSceneDescription):
```

Comment:
it's unfortunate that I'm making a whole new class here, but I couldn't think of a better solution, because the initial position thing below is defined in terms of other dataclass fields